tests/test_server.py

```diff
@@ -48,6 +48,29 @@ def test_drawing_route(self):
         response = self.call_route('/docs/US7654321B2/drawings/1')
         self.assertEqual(200, response.status_code)
 
+    def test_delete_route(self):
+        """ Can Delete File
+        """
+        route = '/docs/US7654321A2'
+        pn = route.split('/')[-1] + '.json'
```

Comment:
These lines can be simplified by starting with the `pn` variable and then creating the `route` variable.